plugins/inputs/statsd/running_stats.go

```diff
@@ -49,7 +49,7 @@ func (rs *RunningStats) AddValue(v float64) {
 	}
 
 	// These are used for the running mean and variance
-	rs.n += 1
+	rs.n++
```

Comment:
avoid this type of code churn

Reply:
Was being flagged by linter.

Reply:
The only static checkers we are using are `go vet` and `go fmt`, You will need to disable any other linters or ignore their errors, since we don't want to update unrelated items like this.

Reply:
ok.